packages/core/src/types/IViewport.ts

```diff
@@ -6,19 +6,112 @@ import { ActorEntry } from './IActor';
 import ViewportType from '../enums/ViewportType';
 import ViewportStatus from '../enums/ViewportStatus';
 import DisplayArea from './displayArea';
+import BoundsLPS from './BoundsLPS';
 
 /**
- * Specifies what image to get a reference for.
+ * Specifies what view to get a reference for.
+ * This set of options allows a Viewport to return a reference for an image
+ * not currently in view, such as for a different slice, or for a given set of
+ * points.
  */
-export type TargetSpecifier = {
+export type ViewReferenceSpecifier = {
   /** The slice index within the current viewport camera to get a reference for */
-  sliceIndex?: number;
-  /** True to get a frame of reference UID reference instead of a regular image one */
+  sliceIndex?: number | [number, number];
+  /**
+   * true to get a frame of reference UID reference instead of a regular image one,
+   * or false to not get it when it would otherwise do so.
+   * The default depends on the view type, but in general showing multiple images
+   * combined, the return will be only for frame of reference, while showing a
+   * single one, it will be for frame of reference.
+   */
   forFrameOfReference?: boolean;
```

Comment:
we can use FrameOfReference here instead of forFOR

Reply:
No, because that isn't a frame of reference, it is indicating to create a view reference that is for a specific frame of reference rather than a specific volume or image.